squarelet/users/managers.py

```diff
@@ -54,10 +54,9 @@ def register_user(self, user_data):
                     )
                 )
 
-            if plan and plan.for_groups:
-                group_organization = Organization.objects.create(
-                    name=user_data["organization_name"]
-                )
+            organization_name = user_data.get("organization_name")
```

Comment:
This change is masking an error somewhere else instead of fixing it - my sign up forms validates that you must supply an organization name if you sign up for a group plan - otherwise there is no organization to apply that group plan to.

Since you are not using my form, your sign up serializer should validate that they do not select a group plan if they do not supply an organization name to create.  I am not sure why you are seeing a key error here - what plan are you trying to sign for when you do that?

Reply:
We've been hardcoding to the default Free plan, which is marked for organizations. My guess is we should make a PressPass free plan that is marked for individuals only.